src/pages/Payment/PaymentProduct.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Payment.scss';
+
+function PaymentProduct({ paylist }) {
```

Comment:
```suggestion
function PaymentProduct({ paylist: { price, id, name, num } }) {
```

이렇게 구조분해할당 해주시면 더 깔끔하게 데이터 관리가 될 것 같아요!